### ISSUE TITLE
annotated class is not found on windows 7

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NickstaDB/reflections#101]
```
What steps will reproduce the problem?
1. Run the AnnotationsTest.java in the attachment.

What is the expected output? What do you see instead?
Test must be successful, annotated class should be found.

What version of the product are you using? On what operating system?
0.9.6, jdk 7u1, windows 7 sp1

Please provide any additional information below.
Try with the patch provided in the attachment, should fix the problem.
```

Original issue reported on code.google.com by `igorewka` on 1 Feb 2012 at 2:31

Attachments:
- [annotations_issue.zip](https://storage.googleapis.com/google-code-attachments/reflections/issue-101/comment-0/annotations_issue.zip)

**Comment by GoogleCodeExporter** [comment]
```
Yes please i have the same issue, im guessing im going to build my own version 
with the patch
```

Original comment by `regis.le...@gmail.com` on 27 Feb 2012 at 6:15

**Comment by GoogleCodeExporter** [comment]
```
I confirm the patch fixes the issue on Windows.
```

Original comment by `Julien.N...@gmail.com` on 3 Apr 2012 at 12:19

**Comment by GoogleCodeExporter** [comment]
```
Issue 104 has been merged into this issue.
```

Original comment by `ronm...@gmail.com` on 2 May 2012 at 3:46

**Comment by GoogleCodeExporter** [comment]
```
fixed on 0.9.7-RC1 and trunk. comments?
```

Original comment by `ronm...@gmail.com` on 2 May 2012 at 3:47
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed 0.9.7
```

Original comment by `ronm...@gmail.com` on 22 May 2012 at 7:47
- Changed state: **Fixed**
